pod/perl5184delta.pod

```diff
@@ -45,12 +45,12 @@ filehandle information is properly reset after a pager is run. [perl #121456]
 =item *
 
 Introduced by
-L<perl #113536|https://rt.perl.org/Public/Bug/Display.html?id=113536>, a memory
+L<[#12161]|https://github.com/Perl/perl5/issues/12161>, a memory
 leak on every call to C<system> and backticks (C< `` >), on most Win32 Perls
 starting from 5.18.0 has been fixed.  The memory leak only occurred if you
 enabled pseudo-fork in your build of Win32 Perl, and were running that build on
 Server 2003 R2 or newer OS.  The leak does not appear on WinXP SP3.
-[L<perl #121676|https://rt.perl.org/Public/Bug/Display.html?id=121676>]
+[L<[#13741]|https://github.com/Perl/perl5/issues/13741>]
```

Comment:
These are already within square brackets, there should only be one set of brackets either inside or outside the link (moving them inside would be consistent with elsewhere).

Reply:
Same with the links in perl5200delta, perl5320delta, perl5320delta

Reply:
Aha, I see, will fix. Thank you.